test/extensions/filters/http/dynamic_forward_proxy/proxy_filter_test.cc

```diff
@@ -90,7 +90,7 @@ TEST_F(ProxyFilterTest, HttpDefaultPort) {
 
   EXPECT_CALL(callbacks_, route());
   EXPECT_CALL(cm_, getThreadLocalCluster(_));
-  EXPECT_CALL(*dns_cache_manager_->dns_cache_, canCreateDnsRequest_(_))
+  EXPECT_CALL(*dns_cache_manager_->dns_cache_, canCreateDnsRequest_())
```

Comment:
ScopedRuntime is no longer needed in this test, I think.

Reply:
fixed